examples/poll/src/components/Modal/AddQuestionModal.spec.tsx

```diff
@@ -0,0 +1,77 @@
+import { VernaForm, VernaProvider } from '@openfun/verna';
+import * as factories from '@openfun/verna/src/tests/mocks/factories';
+import { act, render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { Suspense } from 'react';
+import AddQuestionModal from ':/components/Modal/AddQuestionModal';
+
+describe('feature to add widgets using a modal and a select', () => {
```

Comment:
Please use module name as test suite name.
```suggestion
describe('AddQuestionModal', () => {
```